### PR TITLE
remove old debug feature that is no longer needed.

### PR DESCRIPTION
**Pull request by ZappoMan**

### CODE REVIEW

**Comment by ZappoMan** [comment]
Test plan:
- Smoke test
- verify that `Developer > Network > Disable Entity NACK Packets` is no longer available

**Comment by hifi-gustavo** [comment]
Build console output for commit: 770fab956fac0502b0a95d68f624d20d50bce533 available at: https://gist.github.com/243562bffc5b05587adb10f9e3b0b7e3<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7950.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7950.dmg

File: interface/src/Menu.h [hunk 1]
```diff
@@ -84,7 +84,6 @@ namespace MenuOption {
     const QString DisableActivityLogger = "Disable Activity Logger";
     const QString DisableEyelidAdjustment = "Disable Eyelid Adjustment";
     const QString DisableLightEntities = "Disable Light Entities";
-    const QString DisableNackPackets = "Disable Entity NACK Packets";
```
Comment (Atlante45) [inline interface/src/Menu.h]: You need to remove it from MenuOption.qml as well.

Comment (ZappoMan) [inline interface/src/Menu.h]: Actually MenuOption.qml needs to be deleted completely... it's not in use.

Comment (ZappoMan) [inline interface/src/Menu.h]: fixed

**Comment by Nex-Pro** [comment]
**PR Build:** #7950
**Testing Architecture:** Desktop PC
**CPU:** 4.0 ghz i7
**RAM:** 16 GB
**GPU:** Nvidia gtx770 2gb
**Testing OS:** win 10 pro
**HMD:** None
**Hand Controllers:** None

#### CONFIRMED: **Developer > Network > Disable Entity NACK Packets is no longer available**

**Smoke Test:**
**Testing Locations:**  playa, cellscience, localhost
**QA Tester:** Piper.Peppercorn
**Assisted by:** Michelle Leckrone

**localhost 'createTestBlocks.js' Result**: **localhost 'createTestBlocks.js'** SUCCESS
**localhost ATP Upload + Edit Result**: **localhost ATP Upload + Edit** SUCCESS
**Echo Server Audio**: **Echo Server Audio** SUCCESS
**localhost 'harmonicOscillator.js' Result**: **localhost 'harmonicOscillator.js'** SUCCESS
**localhost 'reverbTest.js' Result**: **localhost 'reverbTest.js'** SUCCESS
**localhost 'playaPerformanceTest.js' Result**: **localhost 'playaPerformanceTest.js'** SUCCESS
**playa 'playaPerformanceTest.js' Result**: **playa 'playaPerformanceTest.js'** SUCCESS
**cellscience 'playaPerformanceTest.js' Result**: **cellscience 'playaPerformanceTest.js'** SUCCESS
**QA Pass**: **QA Pass** SUCCESS

**Domain** |localhost |Playa |cellscience
---|---|---|---
**Render Rate**: |60 |60 |52
**Present Rate**: |60 |60 |60
**Present New Rate**: |60 |60 |52
**CPU Buffer Count**: |5880 |5810 |2805
**GPU Buffer Count**: |1445 |808 |308
**CPU Memory Usage**: |78327808 |106680320 |47640576
**GPU Memory Usage**: |39936000 |33394688 |18333696
**CPU Texture Count**: |364 |383 |171
**GPU Texture Count**: |1445 |1437 |1318
**GPU Transfer Count**: |0 |0 |0
**CPU Texture Memory Usage**: |317302668 |1445820128 |134858872
**GPU Texture Memory Usage**: |1036487762 |775973410 |427946730
**GPU Virtual Memory Usage**: |1260544082|1000029730|652003050
**Triangle Count**: |614012 |407958 |251071
**Triangle Rate**: |38375750 |29139857 |19313153
**API Drawcall Count**: |398 |142 |44
**Drawcall Count**: |6454 |263 |44
**Drawcall Rate**: |403375 |18785 |3384
**State Changes Texture Count**:|299 |163 |22
**Pipeline Count**: |39 |27 |18
**Item Opaques**: |499 |212 |41
**Item Translucents**: |52 |13 |17
**Item Lights**: |29 |26 |0
**Timing Opaques**: |2637 |1393 |373
**Timing Translucents**: |209 |72 |184
**Timing Lighting**: |137 |171 |48
**Render Frame**: |3907 |2328 |1257

**Comment by Nex-Pro** [comment]
@ZappoMan  Do I need to check this again ?

File: interface/src/Menu.h (hunk 1 above)
Comment (Atlante45) [inline interface/src/Menu.h]: Arf, ok.

**Comment by ZappoMan** [comment]
@Nex-Pro - sorry man, but yeah I guess just in case some random piece of code I didn't find is somehow depending on that file, it would be good to basically run it and make sure things don't go south.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 00bd3b2e8a60577f25d081078f4bfe1d98ea7dee available at: https://gist.github.com/243562bffc5b05587adb10f9e3b0b7e3<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7950.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7950.dmg

**Comment by Nex-Pro** [comment]
@ZappoMan  Np man. I'm at the meeting and will get back on it asap

**Comment by Nex-Pro** [comment]
**PR Build:** #7950
**Testing Architecture:** Desktop PC
**CPU:** 4.0 ghz i7
**RAM:** 16 GB
**GPU:** Nvidia gtx770 2gb
**Testing OS:** win 10 pro
**HMD:** None
**Hand Controllers:** None

**CONFIRMED:** ***Menu option - Developer > Network > Disable Entity NACK Packets is no longer available***

**Testing Locations:**  playa, cellscience, localhost

**QA Tester:** Piper.Peppercorn
**QA Second Test**


**localhost 'createTestBlocks.js' Result**: **localhost 'createTestBlocks.js'** SUCCESS
**localhost ATP Upload + Edit Result**: **localhost ATP** SUCCESS
**Echo Server Audio**: **Echo Server Audio** SUCCESS
**localhost 'harmonicOscillator.js' Result**: **localhost 'harmonicOscillator.js'** Only one avatar available.
**localhost 'reverbTest.js' Result**: **localhost 'reverbTest.js'** SUCCESS
**localhost 'playaPerformanceTest.js' Result**: **localhost 'playaPerformanceTest.js'** SUCCESS
**playa 'playaPerformanceTest.js' Result**: **playa 'playaPerformanceTest.js'** SUCCESS
**cellscience 'playaPerformanceTest.js' Result**: **cellscience 'playaPerformanceTest.js'** SUCCESS
**QA Pass**:  SUCCESS


**Domain** |localhost |Playa |cellscience
---|---|---|---
**Render Rate**: |60 |60 |60
**Present Rate**: |60 |60 |60
**Present New Rate**: |60 |60 |60
**CPU Buffer Count**: |4817 |5078 |2711
**GPU Buffer Count**: |625 |473 |287
**CPU Memory Usage**: |60264448 |73367552 |38846464
**GPU Memory Usage**: |12767232 |18808832 |13873152
**CPU Texture Count**: |165 |250 |102
**GPU Texture Count**: |1190 |1281 |1093
**GPU Transfer Count**: |0 |0 |0
**CPU Texture Memory Usage**: |153496616 |942040512 |90370312
**GPU Texture Memory Usage**: |349253913 |328723474 |85284187
**GPU Virtual Memory Usage**: |526953753|506423314|262984027
**Triangle Count**: |209911 |351148 |262727
**Triangle Rate**: |13119437 |20655764 |10946958
**API Drawcall Count**: |182 |111 |47
**Drawcall Count**: |197 |232 |47
**Drawcall Rate**: |12312 |13647 |1958
**State Changes Texture Count**:|146 |116 |22
**Pipeline Count**: |26 |22 |14
**Item Opaques**: |225 |158 |42
**Item Translucents**: |9 |7 |17
**Item Lights**: |22 |23 |0
**Timing Opaques**: |1244 |3116 |1208
**Timing Translucents**: |87 |194 |1190
**Timing Lighting**: |118 |520 |163
**Render Frame**: |2088 |6005 |5542

**Comment by hifi-gustavo** [comment]
Build console output for commit: ac49feda228ee07c4d1c542121ca5ed31d7c864b available at: https://gist.github.com/243562bffc5b05587adb10f9e3b0b7e3<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7950.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7950.dmg

**Pull request merged**